website/docs/core-concepts/workflows/workflows.md

```diff
@@ -21,34 +21,119 @@ workflows:
+# and 'workflows.base_path' are independent settings (supporting both absolute and relative paths).
+# If 'base_path' is provided, 'components.terraform.base_path', 'components.helmfile.base_path', 'stacks.base_path' 
+# and 'workflows.base_path' are considered paths relative to 'base_path'.
+base_path: ""
+
+workflows:
+  # Can also be set using 'ATMOS_WORKFLOWS_BASE_PATH' ENV var, or '--workflows-dir' command-line arguments
+  # Supports both absolute and relative paths
+  base_path: "stacks/workflows"
 ```
 
-**Note:** Workflows defined in the stack config files can be executed only for the particular stack (environment and stage). It's not possible to
-provision resources for multiple stacks this way.
+- `base_path` - the base path for components, stacks and workflows configurations
+
+- `workflows.base_path` - the base path to Atmos workflow files
+
+### Create Workflow Files
+
+In `atmos.yaml`, we set `workflows.base_path` to `stacks/workflows`. The folder is relative to the root of the repository.
+
+Refer to [workflow1.yaml](https://github.com/cloudposse/atmos/tree/master/examples/complete/stacks/workflows/workflow1.yaml) as an example.
+
+We put the workflow files into the folder. The workflow file names can be anything you want, but we recommend naming them according to the functions
+they are performing, e.g. create separate workflow files per environment, account, team, or service.
+
+For example, you can have a workflow file `stacks/workflows/workflows-eks.yaml` to define all EKS-related workflows.
+
+Or, you can have a workflow file `stacks/workflows/workflows-dev.yaml` to define all workflows to provision resources into the `dev` account.
+Similarly, you can create a workflow file `stacks/workflows/workflows-prod.yaml` to define all workflows to provision resources into the `prod`
+account.
+
+You can segregate the workflow files even further per account and service. For example, in the workflow
+file `stacks/workflows/workflows-dev-eks.yaml` you can define all EKS-related workflows for the `dev` account.
+
+Workflow files must confirm to the following schema:
+
+```yaml
+workflows:
+
+  workflow-1:
+    description: ""
+    steps: [ ]
+
+  workflow-2:
+    description: ""
+    steps: [ ]
+```
+
+Each file must have the `workflows:` top-level section with a map of workflow definitions.
+
+### Workflow Schema
+
+Each workflow definition must confirm to the following schema:
+
+```yaml
+workflows:
+
+  workflow-1:
+    description: ""
+    steps: [ ]
+
+  workflow-2:
+    description: ""
+    steps: [ ]
+```
+
+:::note
+
+A workflow command of type `shell` can be any simple or complex shell command or script
 
-In the second case (defining workflows in a separate file), a single workflow can be created to provision resources into different stacks. The stacks
-for the workflow steps can be specified in the workflow config.
+:::
 
-## Using Workflow Files
+## Workflow Examples
```

Comment:
We're missing the explanation of how to use shell commands. Ideally, using parameters.